ols/src/llms/utils.py

```diff
@@ -0,0 +1,42 @@
+from langchain.callbacks.base import BaseCallbackHandler
+from langchain_core.outputs.llm_result import LLMResult
+from typing import Any, Dict, List
+import logging
+
+logger = logging.getLogger(__name__)
+
+class GenericTokenCounter(BaseCallbackHandler):
```

Comment:
Do you plan to add a docstring here? Example usage might be useful to see here.

Reply:
yes, i should have added a docstring here. will add in new commit